generators/boilerplate/index.js

```diff
@@ -6,6 +6,14 @@ module.exports = generators.Base.extend({
   constructor: function () {
     generators.Base.apply(this, arguments);
 
+    // destination path
+    this.option('path', {
```

Comment:
`path` is pretty vague. Maybe something like root or project root would be more descriptive?